app/src/main/java/de/westnordost/streetcomplete/quests/hairdresser/AddHairdresser.kt

```diff
@@ -0,0 +1,46 @@
+package de.westnordost.streetcomplete.quests.hairdresser
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddHairdresser : OsmFilterQuestType<Hairdresser>() {
```

Comment:
Hm, the name doesn't really fit. The quest is not about adding hairdressers.

Reply:
ok, renamed to HairdresserCustomers instead